relocate.html

```diff
@@ -34,7 +34,7 @@ <h2 class="preference">We give you the power to choose your preference</h2><br><
             </p>
         </div>
         <div class="lorry">
-            <img src="images/maxim-tolchinskiy-crHhZlES310-unsplash.jpg" alt="lorry" width="300" height="300">
```

Comment:
link the correct images. some images used do do not exist in in our "assets" file